server/app.js

```diff
@@ -28,7 +29,21 @@ const io = socketio(server, {
 });
 
 io.on("connection", (socket) => {
-  console.log("connected");
+  socket.on('JoinConversation', ({ userProfileId,conversationId },cb) => {
+    const{error}=addUser({id:socket.id,userProfileId,conversationId})
```

Comment:
We should probably set up a code formatter and be cautious of this going forward for more readable and maintainable code. We use Prettier for this project. Thanks!

Reply:
Please format this whole change here to match the project as well.  

https://www.digitalocean.com/community/tutorials/code-formatting-with-prettier-in-visual-studio-code#:~:text=In%20the%20command%20palette%2C%20search,Then%20choose%20Prettier%20%2D%20Code%20Formatter.

Reply:
Please fix the formatting here. Thanks!